src/Accounts/Authentication/KeyStore/AzKeyStore.cs

```diff
@@ -72,7 +72,7 @@ private void UpdateStorage(KeyStoreNotificationArgs args)
             helper.WriteToCachedStorage(args.KeyCache);
         }
 
-        public void Clear()
+        public virtual void Clear()
```

Comment:
Could you please explain here? Who is inheriting current class?

Reply:
To support Mock test. As the class is not used outside Az.Accounts, it is safe to change its definition.